source/TR-181/middle_layer_src/gponmgr_dml_hal_param.c

```diff
@@ -253,12 +253,12 @@ ANSC_STATUS Map_hal_dml_pm(DML_PHY_MEDIA_LIST_T* gponPhyList, char* ParamName, c
     }
     else if( strstr(ParamName, "NominalBitRateDownstream"))
     {
-        pPhyMedia->NominalBitRateDownstream = strtoul(pValue,&err,10);
+        pPhyMedia->NominalBitRateDownstream = atoll(pValue);
```

Comment:
Why we are replacing strtoul for atoll. Atoll is deprecated.

Reply:
changed from atoll to strtoull